package.json

```diff
@@ -71,13 +71,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "yarn run lint:fix"
```

Comment:
I had to change that because it failed otherwise, even though `yarn lint` ran without errors